### DISCUSSION TITLE
Navigation appearance with highest contrast

### DISCUSSION THREAD

**Discussion by RHoffi** [devemux86/cruiser#128] [Q&A] [answered]
I am trying to find out which color setting provide the highest contrast and is good readable on sunlight.
Has someone already found out a good working color setting?

Thx in advance

**Comment by devemux86** [comment]
Hi @RHoffi,

Do you search for a map theme or do you mean the options in "Settings | Map | Appearance | Map color"?

<img src="https://user-images.githubusercontent.com/3484020/232202919-8bd5f85b-44da-4de2-8fff-c42d8ec0addc.jpg" width="33%"/>

**Reply by RHoffi** [reply]
Sorry forgot to mention, that i mean the colors for the Infopanels( Background/Foreground) under "Application-Appearance" .
I am trying to find out good color-combinations

**Comment by devemux86** [comment]
It also depends on whether you want a light or dark background, e.g. with an OLED screen.

Here are 3 examples with backgrounds light | transparent | dark:

<img src="https://user-images.githubusercontent.com/3484020/232206622-be3217cb-6af6-4606-be73-9e6643cc83f0.jpg" width="33%"/><img src="https://user-images.githubusercontent.com/3484020/232206625-f3f53205-f51d-47d7-a690-c72df6d4d2f2.jpg" width="33%"/><img src="https://user-images.githubusercontent.com/3484020/232206634-b6da276e-1e81-4204-a89f-9be3ffdce2dd.jpg" width="33%"/>

**Comment by WolfgangZ42** [comment, marked as answer]
For my taste, colors are more distracting than helpful. I drive for years with "black and white" and appropriate transparency. Since recently with the map style from @MotoUKRider:
<img width="400" src="https://user-images.githubusercontent.com/121806238/232246997-ebdd8979-fb8f-4e50-b63b-e25464e41d1e.png"/>

**Comment by MotoUKRider** [comment]
My current look is a solid "soft grey" Display Colour (A:255,R:231,G:238,B:242) which works well for both day+night use, example:

![navigation-example](https://user-images.githubusercontent.com/125696109/232250621-ebc8ba47-1cd9-44a1-a8d8-ce546a0a5fcb.jpg)

It's worth considering the phone's ability to produce a decent brightness in daylight and more so its position on the bike: Above the instrument-cluster (behind the screen, if any) at a slight angle would be preferable to having the phone down on the bars pointing up at 45-degrees where it'll almost certainly produce glare in the direction you happen to be riding!

**Comment by RHoffi** [comment]
Thanks for your examples. Black and white seems to be the best solution.
